src/app/story/story.component.ts

```diff
@@ -79,6 +82,10 @@ export class StoryComponent implements OnInit, AfterViewInit, OnDestroy {
         const nodeDisplayNamePipe = new NodeDisplayNamePipe();
         const titleDisplay = nodeDisplayNamePipe.transform(this.selectedNode.display_name);
         this.titleService.setTitle(titleDisplay);
+        this.processingTimeFlag = story.meta.processing_time_flag;
+        if (this.processingTimeFlag === true) {
+          this.notification.warning('Not enough information to accurately calculate processing time');
```

Comment:
How about `There wasn't enough information to accurately calculate the processing time`? It sounds less like programmer speak.  :stuck_out_tongue: